operator/bundle_helpers/patch-csv.py

```diff
@@ -47,7 +47,7 @@ def must_replace_suffix(str, suffix, replacement):
     return splits[0] + replacement
 
 
-def patch_csv(csv_doc, version, operator_image, first_version, no_related_images, rbac_proxy_replacements):
+def patch_csv(csv_doc, version, operator_image, first_version, no_related_images, downstream_arches, rbac_proxy_replacements):
```

Comment:
While we use this script downstream, we typically try to keep the entire downstream terminology out of the upstream repo (and should keep doing that). Therefore, I would ask you to avoid (a) a `downstream` variable/parameter/flag name and (b) an assumption about what happens downstream. I leave the changes I would suggest throughout the PR as extra comments.